lib/components/clock/clock.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import * as actions from '../../actions/clock';
+
+export class Clock extends React.Component {
+    componentDidMount() {
+        this.props.getTime();
```

Comment:
As far as I know, there is no immutability that can be done from the `component -> action creator -> store` until it meets the reducers? All mechanisms to limit the number of operators on data through immutability/value checks all get done on the other side of the equation? 

To bring it back to my initial point, would the first value of time/date not be defined as part of the store's initial state, upon initialization? What's the benefit of having this set on the component side of things?